TESTING.md

```diff
@@ -249,6 +249,54 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+
+First, an initial management cluster needs to be created using [`EKS Anywhere`](https://anywhere.eks.amazonaws.com/docs/getting-started/production-environment/vsphere-getstarted/#create-an-initial-cluster).
+You can then set `TESTSYS_MGMT_CLUSTER_KUBECONFIG` to the path to the management clusters kubeconfig.
+You need to [build](BUILDING.md) Bottlerocket and a publicly accessible [TUF repository](https://github.com/bottlerocket-os/bottlerocket/blob/develop/PUBLISHING.md#repo-location) to test VMware variants.
+Either `Infra.toml` or your environment need to be configured.
+Make sure to set: GOVC_URL, GOVC_USERNAME, GOVC_PASSWORD, GOVC_DATACENTER, GOVC_DATASTORE, GOVC_NETWORK, GOVC_RESOURCE_POOL, GOVC_FOLDER.
+If `Infra.toml` is present, TestSys will use the data center specified in `Test.toml` first.
+If no data center is specified in `Test.toml`, then TestSys will use the first data center listed in `Infra.toml`.
```

Comment:
The first line in the docs is a little bit "unusual", I read it as "if `Infra.toml` exists it will be ignored since `Test.toml` has higher precedence". You can rearrange the two sentences and avoid a few words to express the same precedence:

```suggestion
TestSys will use the data center speficied in `Test.toml` first.
If no data center is specified in `Test.toml`, Testsyswill use the first data center listed in `Infra.toml`
```